src/index.html

```diff
@@ -5,8 +5,409 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+            href="#"
+            class="header-link"
+            data-qa="nav-hover">
+              Laptops & Computers
+            </a>
```

Comment:
You have to format this markup
```suggestion
            <a
                href="#"
                class="header-link"
                data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```